### PR TITLE
Literally just sets the images to links

### PR DESCRIPTION
**Pull request by Dr-N0**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,32 +94,40 @@ <h2 class="header">Things we've done</h2>
...
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" alt="Two of CSH's networked vending machines, Drink and Snack.">
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/WebDrink-2.0">
```
Comment (RamZallan) [inline index.html]: This isn't the current webdrink code ([https://github.com/ramzallan/tonic](https://github.com/ramzallan/tonic)).

File: index.html [hunk 2]
```diff
@@ -94,32 +94,40 @@ <h2 class="header">Things we've done</h2>
...
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" alt="Harold, a box with a sensor on it, that plays members' music" />
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/Harold">
```
Comment (RamZallan) [inline index.html]: Similarly, I thought there was a new implementation of HAROLD being made (by Nate maybe?)

File: index.html [hunk 3]
```diff
@@ -94,32 +94,40 @@ <h2 class="header">Things we've done</h2>
...
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" alt="Two custom arcade machines in the CSH library" />
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/Arcade-Controls">
```
Comment (RamZallan) [inline index.html]: Would love some confirmation by Brandon Chiu/R&D/anyone w/ knowledge of the arcade cabinets if this is still in use, as the repo hasn't been touched in 10 years.

File: index.html (hunk 1 above)
Comment (devinmatte) [inline index.html]: This is not the current repo. Although the current ones are not under the ComputerScienceHouse org, so not sure what to link

Comment (RamZallan) [reply index.html]: Jinx!

File: _includes/footer.html [hunk 4]
```diff
@@ -46,9 +46,9 @@
     </div>
 </footer>
 
-<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
```
Comment (devinmatte) [inline _includes/footer.html]: Separate PR Please

Comment (Dr-N0) [reply _includes/footer.html]: Hol up, github being fucky on my phone

File: index.html (hunk 2 above)
Comment (Dr-N0) [reply index.html]: It's not a CSH repo

**Pull request closed without merging**